public_root_of_trust.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+	"time"
+
+	sigtuf "github.com/sigstore/sigstore/pkg/tuf"
+	tufclient "github.com/theupdateframework/go-tuf/client"
+)
+
+var globalBuffer bytes.Buffer
+
+type inMemoryDest struct{}
+
+func (d inMemoryDest) Write(p []byte) (n int, err error) {
+	return globalBuffer.Write(p)
+}
+
+func (d inMemoryDest) Delete() error {
+	panic("inMemoryDest delete function should not run")
+}
+
+func GetTargets(usage sigtuf.UsageKind, proxy Proxy) ([]sigtuf.TargetFile, error) {
+	// client initialization
+	httpClient := &http.Client{
+		Timeout: 20 * time.Second,
```

Comment:
InsecureSkipVerify: true,
is not specified

Reply:
This has been updated.